docs/index.rst

```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
       helm list
     NAME                REVISION    UPDATED                     STATUS      CHART                   NAMESPACE
     my-confluent-oss    1           Tue Jun 12 16:56:39 2018    DEPLOYED    cp-helm-charts-0.1.0    default 
+Persistence
+~~~~~~~~~~~~~~~~~~~
+The ZooKeeper and Kafka cluster are deployed with ``StatefulSets`` that have a ``volumeClaimTemplate`` which provides the persistent volume for each replica. You can define the size of the volumes by changing ``dataDirSize`` and ``dataLogDirSize`` under ``cp-zookeeper`` and ``size`` under  ``cp-kafka`` in `values.yaml <https://github.com/confluentinc/cp-helm-charts/blob/master/values.yaml>`__
+
+You also could use the cloud provider's volumes by specifying `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/>`__.. For example, if you are on AWS your storage class will look like this:
+
+.. code:: yaml
+
+      apiVersion: storage.k8s.io/v1beta1
+      kind: StorageClass
+      metadata:
+        name: ssd
+      provisioner: kubernetes.io/aws-ebs
+      parameters:
+        type: gp2
+
+.. tip:: To adapt this example to your needs, read the Kubernetes `StorageClass <https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters>`__ documentation.
+
+The ``StorageClass`` that was created can be specified in ``dataLogDirStorageClass`` and ``dataDirStorageClass`` under 
```

Comment:
don't break line.
modern editors support word wrapping.
also this https://github.com/brandon-rhodes/blog/blob/master/texts/brandon/2012/one-sentence-per-line.rst